### PR TITLE
Grid : Set column with Conf

### PR DESCRIPTION
**Pull request by romfabbro**
https://www.pivotaltracker.com/story/show/132576533

### CODE REVIEW

**Comment by Matheo13** [comment]
SSMS:
Msg 207, Niveau 16, État 1, Ligne 15
Nom de colonne non valide : 'ColumnParams'.

**Comment by romfabbro** [comment]
you have an sql pipe to pass which tagged by "db update"

**Comment by romfabbro** [comment]
see the only one commit 
thank you

**Comment by Matheo13** [comment]
Ok I had to launch first 

ALTER Table ModuleGrids 
ADD ColumnParams varchar(250)

then the rest.

I'll test it right now

File: Front/app/ns_modules/ns_grid/grid.view.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ define([
         onGridReady: function(){
           $.when(_this.deferred).then(function(){
             setTimeout(function(){
-              _this.gridOptions.api.sizeColumnsToFit();
+              //_this.gridOptions.api.sizeColumnsToFit();
```
Comment (Matheo13) [inline Front/app/ns_modules/ns_grid/grid.view.js]: Don't comment it. It's used somewhere else. I don't have rights on your branch.
Column's size must be defined in base, then auto fit from those sizes.

**Review by Matheo13 — CHANGES_REQUESTED** [review]

**Comment by Matheo13** [comment]
Then it's fine.

File: Front/app/ns_modules/ns_grid/grid.view.js (hunk 1 above)
Comment (romfabbro) [reply Front/app/ns_modules/ns_grid/grid.view.js]: done

**Comment by Matheo13** [comment]
minWidth to 100 insted of 50 in wouldn't be better?

**Review by Matheo13 — APPROVED** [review]

**Pull request merged**